temp/automatic_ticket_revocations_proposal.md

```diff
@@ -0,0 +1,123 @@
+# Proposal Structure
+
+The structure of this proposal follows the pattern established by previously proposed consensus changes.  There are two main sections of this proposal:
+
+- General Audience
+- Technical Details
+
+The general audience section provides a high-level overview of the proposal.  It includes details suitable for all stakeholders, including those unfamiliar with the inner workings of consensus.
+
+The technical details section contains a more in-depth discussion of the relevant technical aspects, implementation specifics, and rationale for the chosen semantics of the proposal.
+
+# General Audience
+
+## High-Level Overview
+
+This proposal intends to significantly improve the stakeholder user experience around ticket revocations by automatically revoking missed or expired tickets.
+
+Further, it will enable the recovery of funds for users who lost their redeem script for the legacy VSP system (before the release of [vspd](https://blog.decred.org/2020/06/02/A-More-Private-Way-to-Stake/), which removed the need for the redeem script).
+
+## Current State of Ticket Revocations
+
+Missed and expired tickets must be revoked to return the funds used to purchase the ticket to the user.  Missed and expired tickets are defined as follows:
+
+- **Missed tickets** are tickets that fail to cast a vote when selected.  Tickets are pseudorandomly selected to vote for each block.  If the wallet with voting rights for that ticket is not online to cast its vote when a ticket is selected to vote, the ticket is marked as missed.
+
+- **Expired tickets** are tickets that are not selected to vote before the ticket expiration duration (40,960 blocks, or approximately 142 days, on mainnet).
+
+Revoking a missed or expired ticket requires broadcasting a **ticket revocation transaction**.  In order to manually revoke a ticket today, a user needs to:
+- Open the wallet that has voting rights for the ticket so that the ticket revocation transaction can be signed
+- If the wallet is running in SPV mode, the user will need to disable SPV, sync the blockchain, and rescan
```

Comment:
This isn't strictly true because it's not really required.  It is correct that the current release version of Decrediton requires this, but that is because there isn't a good way for it to know when a ticket has been missed in order to only conditionally show the revoke button when one or more tickets are missed.  However, there is no reason it has to work that way, and, I'm actually fairly certain next Decrediton release is changing this by adding a button which will just have no effect when a ticket is not eligible to be revoked.

In other words, I would probably remove this as it isn't actually a requirement imposed by the existing consensus rules.  Alternatively, you could include something similar, but if you decide to, I think it should call out the actual root cause stemming from the consensus rules and the resulting poor user experience that occurs as a result.

Reply:
Ya, good point that it only applies to the current version of Decredition.  I just went ahead and removed it as suggested.